[Feature-7465][Alert] Modifiy alert docs (#594)
apache/dolphinscheduler-website: docs/en-us/dev/user_doc/guide/alert/enterprise-wechat.md

# Enterprise WeChat

> If you need to use Enterprise WeChat to alert, please create alarm Instance in warning instance manage, and then chose the wechat plugin. The configuration example of enterprise WeChat is as follows

Reviewer: ```suggestion
If you need to use Enterprise WeChat to alert, please create an alarm Instance in warning instance manage, and then choose the wechat plugin. The configuration example of enterprise WeChat is as follows
```

> <img src="enterprise-wechat-plugin.png">

Reviewer: I thinks we should use markdown syntax here, `[enterprise-wechat-plugin](enterprise-wechat-plugin.png)`. WDYT
```suggestion
<img src="enterprise-wechat-plugin.png">
```

> Where send Type corresponds to app and appchat respectively:

Reviewer: ```suggestion
Where send type corresponds to app and appchat respectively:
```

> APP: https://work.weixin.qq.com/api/doc/90000/90135/90236

Reviewer: Is there a documnet in English? I think English is better but I could not find it?

Author: https://developers.weixin.qq.com/community/develop/doc/000ea2cdb84d9009c9ea7c7b651400
There is no English version of enterprise wechat API documents.

Reviewer: Ok, thanks for the clarification 👍 . But I still want to 👎  for wechat enterprise API doc
